calico/reference/felix/configuration.md

```diff
@@ -169,6 +169,7 @@ See the [HOWTO guide]({{ site.baseurl }}/maintenance/ebpf/enabling-ebpf) for ste
 | BPFMapSizeIPSets / <br/> FELIX_BPFMapSizeIPSets | Controls the size of the IPSets map. The IP sets map must be large enough to hold an entry for each endpoint matched by every selector in the source/destination matches in network policy.  Selectors such as "all()" can result in large numbers of entries (one entry per endpoint in that case). | int | 1048576 |
 | BPFMapSizeRoute / <br/> FELIX_BPFMapSizeRoute | Controls the size of the route map. The routes map should be large enough to hold one entry per workload and a handful of entries per host (enough to cover its own IPs and tunnel IPs). | int | 262144 |
 | BPFHostConntrackBypass / <br/> FELIX_BPFHostConntrackBypass | Controls whether to bypass Linux conntrack in BPF mode for workloads and services. | true,false | true |
+| BPFPolicyDebugEnabled  / <br/> FELIX_BPFPOLICYDEBUGENABLED    | In eBPF dataplane mode, controls whether felix will collect policy dump for each interface. | true, false |  true |
```

Comment:
Same comment here, it's true but it'd be a lot more valuable to the user if you gave them a signpost on how to access the data.